apps/docs/docs/core/core.md

```diff
@@ -165,6 +168,38 @@ export const contract = c.router({
 });
 ```
 
+### Schema validation on the client
+
+By default, all responses are inferred at the type-level by the client using the contract, and are not validated at runtime.
+
+However, you can use the `validateResponseOnClient` option to validate the response at runtime by checking it against the defined schema associated with the status code in the contract. By default, this option is set to `false`.
+
+If you would like to enable this functionality for all routes in the contract, you can set the `validateResponseOnClient` option to `true` when initializing the contract.
+
+```typescript
+const c = initContract();
+export const contract = c.router({
+  {
+    ... // endpoints
+  },
+  {
+    validateResponseOnClient: true,
+  }
+});
+```
+
+You can also control this option on a per-route basis which will also override the globally set option.
+
+```typescript
+const c = initContract();
+export const contract = c.router({
+  getPosts: {
+    ...,
+    validateResponseOnClient: true,
+  }
+});
+```
```

Comment:
Awesome! Cheers 🍾

Reply:
I seem to have made a teeny tiny oopsie here.

My caution is a sub-heading down. Fixing now.